src/Video-detail/notes-container.js

```diff
@@ -0,0 +1,133 @@
+import { getNoteDetails } from "../utils";
+import { v4 as uuidv4 } from 'uuid';
+
```

Comment:
Contents of this file could be simplified by refactoring the components as new files. (since this file is named as notes-container, i was expecting only container details. However, this has all the implementation w.r.t notes)

Reply:
Yeah i will create separate folder for notes and create files for each component related to notes